actions/promote/worker.js

```diff
@@ -179,7 +179,7 @@ async function promoteFloodgatedFiles(doPublish, batchManager, appConfig) {
     let previewStatuses = [];
     let publishStatuses = [];
     if (ENABLE_HLX_PREVIEW) {
-        previewStatuses = await previewOrPublishPages(PREVIEW);
+        previewStatuses = await bulkPreviewOrPublish(PREVIEW);
```

Comment:
There is another call happening for PUBLISH. I would assume, that should call the new API as well?
https://github.com/adobecom/milo-fg/blob/b458ed9b498ec3251c3e6ce647ecb5a4d9f2b13d/actions/promote/worker.js#L189

Reply:
Thank you @sukamat . As suggested updated code with bulk publish api.